### PR TITLE
Update header links

### LINKED ISSUE #27: Website header links "GitHub" and "About" are nearly identical
### Kind of issue 

- [x] Improvement
- [ ] Bug
- [ ] Other, namely:

### Description

Currently the _GitHub_ link on the website links to:

```
https://github.com/simple-icons/simple-icons
```

And the _About_ link on the website links to:

```
https://github.com/simple-icons/simple-icons#readme
```

The difference being:

```diff
- https://github.com/simple-icons/simple-icons
+ https://github.com/simple-icons/simple-icons#readme
```

I think it's worth considering removing one of the two. My suggestion would be to remove the _About_ link and possibly change the GitHub link to:

```
https://github.com/simple-icons/simple-icons#readme
```

Discussion and suggestions are welcome 🙂

### LINKED ISSUE #59: Add link to CDNs
### Kind of issue 

- [x] Feature
- [ ] Bug
- [ ] Design
- [ ] Other, namely:

### Description

Currently the website has external links to GitHub, NPM, and Packagist. We should also add link for - at least - the CDNs we promote in [the documentation](https://github.com/simple-icons/simple-icons#cdn-usage).

### PR DESCRIPTION
**Pull request by ericcornelissen**
- Closes #27: The "About" link has been removed.
- Closes #59: Links to both JSDeliver and Unpkg have been added.

### Preview

| Before | After |
| --- | --- |
| preview links after | preview links after |

### CODE REVIEW

**Review by adamrusted — APPROVED** [review]
Looks good to me! Will wait on @PeterShaggyNoble's approval though, as you asked him to review 👍

**Review by PeterShaggyNoble — APPROVED** [review]
Looks good to me, although I'd still personally prefer we use icons.

**Comment by ericcornelissen** [comment]
> Looks good to me, although I'd still personally prefer we use icons.

Agreed, but just replacing this by icons looks a bit off in my opinion so I stuck with the current approach for now and we can change that later :+1:

**Pull request merged**
